docs/content/guides/dagster/memoization.mdx

```diff
@@ -4,170 +4,79 @@ title: Versioning and Memoization | Dagster
 
 # Versioning and Memoization <Experimental />
 
-<CodeReferenceLink filePath="examples/memoized_development" />
+<CodeReferenceLink filePath="examples/docs_snippets/docs_snippets/guides/dagster/versioning_memoization" />
 
-This example is intended to describe how to use Dagster's versioning and memoization features. As of release `0.10.0`, Dagster has functionality for tagging a solid or resource with the _version_ argument of the decorator (ie `@solid(version="a")`, `@resource(version="b")`). This version argument is meant to be a representation of the state of the code contained by the wrapper.
+This example describes how to use Dagster's versioning and memoization features.
 
-Dagster can use versions to determine whether or not it is necessary to re-execute a particular step. Given versions of the code from each solid in a pipeline, the system can infer whether an upcoming execution of a step will differ from previous executions by tagging solid outputs with a version. This allows for the outputs of previous runs to be re-used. We call this process **memoized execution**.
```

Comment:
I think it's worthwhile to retain this high-level overview of what memoization / versioning is at the top.